java/ql/test/experimental/query-tests/security/CWE-927/IntentSources.java

```diff
@@ -0,0 +1,37 @@
+package com.example.myapp;
+
+import android.app.Activity;
+
+public class IntentSources extends Activity {
+
+	public void test() {
+
+		String trouble = this.getIntent().getStringExtra("key");
+		Runtime.getRuntime().exec(trouble);
```

Comment:
The test case in this dir looks more like a library test than a query test, and there's no explicit CWE-927 query to test. Also, the spurious relation to `ExecTainted.ql` is unnecessary.
Could you please move this test case to `java/ql/test/library-tests/dataflow/taintsources`? That dir already has a `.ql` file set up to test the various `RemoteFlowSource`s, so it'll just be a matter of augmenting its `.expected` and `options` files once the `.java` and manifest files are moved there.

Reply:
I've moved this test case (<code>IntentSources.java</code> and <code>AndroidManifest.xml</code>) to <code>java/ql/test/library-tests/dataflow/taintsources/</code> and updated <code>java/ql/test/library-tests/dataflow/taintsources/options</code> and <code>java/ql/test/library-tests/dataflow/taintsources/remote.expected</code>